content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx

```diff
@@ -108,29 +109,29 @@ Note that the Upgrade CLI is designed in a way that **you can stop and re-start
 
 There are two main upgrade strategies:
 
-- **Upgrade all at once**: Entirely remove Prisma 1 from your project and move everything over to Prisma 2 at once.
-- **Gradual upgrade side-by-side**: Add Prisma 2 to the existing Prisma 1 project and gradually replace existing Prisma 1 features with Prisma 2 while running them side-by-side.
+- **Upgrade all at once**: Entirely remove Prisma 1 from your project and move everything over to Prisma 2+ at once.
+- **Gradual upgrade side-by-side**: Add Prisma 2+ to the existing Prisma 1 project and gradually replace existing Prisma 1 features with Prisma 2+ while running them side-by-side.
 
-Note that if you are planning to run Prisma 1 and Prisma 2 side-by-side, you must not yet resolve the [schema compatibilities](#schema-incompatibilities) that are breaking the Prisma 1 setup.
+Note that if you are planning to run Prisma 1 and Prisma 2+ side-by-side, you must not yet resolve the [schema compatibilities](#schema-incompatibilities) that are breaking the Prisma 1 setup.
 
 ### When to choose which strategy
 
 If your project is not yet running in production or has little traffic and user data, the **all at once** strategy is recommended.
 
-In case your project already sees a lot of traffic and has a lot of user data stored in the database, you might want to consider the **gradual** upgrade strategy where you're running Prisma 1 and Prisma 2 side-by-side for a certain amount of time until you've replace all former Prisma 1 functionality with Prisma 2
+In case your project already sees a lot of traffic and has a lot of user data stored in the database, you might want to consider the **gradual** upgrade strategy where you're running Prisma 1 and Prisma 2+ side-by-side for a certain amount of time until you've replace all former Prisma 1 functionality with Prisma 2+
 
-Note that you won't be able to fix the [schema incompatibilities](#schema-incompatibilities) that require are breaking Prisma 1 if you choose the gradual upgrade strategy and intend to run Prisma 1 and Prisma 2 side-by-side. That's because these data migrations are breaking the schema that Prisma 1 expects. This means that your Prisma Client API might not feel as idiomatic as it could, but you still get the full feature set of Prisma Client.
+Note that you won't be able to fix the [schema incompatibilities](#schema-incompatibilities) that require a "Breaking Prisma 1" change if you choose the gradual upgrade strategy and intend to run Prisma 1 and Prisma 2+ side-by-side. That's because these data migrations are breaking the schema that Prisma 1 expects. This means that your Prisma Client API might not feel as idiomatic as it could, but you still get the full feature set of Prisma Client.
```

Comment:
Unrelated fix to make the sentence to make a bit more sense. Still not optimal, but should be better at least.